how-it-works/3_governance/01_network-nervous-system-nns.subpage.md

```diff
@@ -5,27 +5,91 @@ shareImage: /img/how-it-works/network-nervous-system.webp
 slug: network-nervous-system-nns
 ---
 
-# Network Nervous System: Internet Computer Blockchain Governance
+# The Network Nervous System (NNS)
 
 [![Watch youtube video](https://i.ytimg.com/vi/hWnsluxmRqc/maxresdefault.jpg)](https://www.youtube.com/watch?v=hWnsluxmRqc)
 
-The Network Nervous System (NNS) is the open algorithmic governance of the Internet Computer blockchain. Its most notable innovations include its ability to upgrade the Internet Computer protocol and software running on node machines, onboard new node providers, add node machines into the blockchain network, and create new subnet blockchains to increase capacity. Anyone can participate in the NNS by staking ICP tokens into one or more neurons. Neurons can submit proposals and decide to adopt or reject them. The NNS implements liquid democracy: neurons can follow other neurons a delegate voting power.
 
+The Network Nervous System (NNS) is the DAO that controls the Internet Computer blockchain.
+It is an open, permissionless governance system, where everyone can participate by staking ICP utility tokens in so-called _neurons_.
+Decisions are made by voting on _proposals_, which can be submitted by anyone.
+
+
+
+
+## Governance participation
+Governance participants are called _neurons_.
+Neurons are created by locking ICP utility tokens for a specified period of time.
+The NNS is a stake-based governance system, where a neuron's voting power is depending on three main factors:
+1) the amount of ICP tokens the neuron stakes
+2) the neuron's _dissolve delay_ which is the duration of the staking period
+3) the neuron's _age_, which is the time since the tokens have been staked.
```

Comment:
More precisely this would be "the neuron's _age_, which is the time since the tokens have been staked in a non-dissolving neuron."

Reply:
Yes. I wanted to omit the fact that a neuron can be dissolving and non-dissolving as explaining this would require more space and I think we should keep it high level here.. 
Let me change it to "which is on a high level" to indicate that this is not fully precise but still not explain in more detail. Hope taht make sense.